pkg/debug/scripts/collect-info.sh

```diff
@@ -265,7 +265,7 @@ chroot /hostfs lsusb -vvv -t > "$DIR/lsusb-vvv-t"
     ls -l /sys/class/net/
 } > "$DIR/sys-fs-usb"
 
-dmesg            > "$DIR/dmesg"
+dmesg -T            > "$DIR/dmesg-T"
```

Comment:
It looks like there is an extra TAB in the indentation....

Reply:
Ooops, fixed